planning/behavior_path_planner/autoware_behavior_path_static_obstacle_avoidance_module/README.md

```diff
@@ -257,14 +257,14 @@ The module generates detection area for target filtering based on following para
 
 ##### Width of detection area
 
-1. get the largest lateral margin of all classes (Car, Truck, ...). The margin is sum of `soft_margin` and `hard_margin_for_parked_vehicle`.
-2. the detection area width is sum of ego vehicle width and the largest lateral margin.
+1. Get the largest lateral margin of all classes (Car, Truck, ...). The margin is the sum of `soft_margin` and `hard_margin_for_parked_vehicle`.
+2. The detection area width is the sum of ego vehicle width and the largest lateral margin.
 
 ##### Longitudinal distance of detection area
 
 If the parameter `detection_area.static` is set to `true`, the module creates detection area whose longitudinal distance is `max_forward_distance`.
 
-If the parameter `detection_area.static` is set to `false`, the module creates detection area so that the ego can avoid objects with minimum lateral jerk value. Thus, the longitudinal distance is depends on lateral maximum shift length, lateral jerk constraints and current ego speed. Additionally, it has to consider distance used for prepare phase.
+If the parameter `detection_area.static` is set to `false`, the module creates detection area so that the ego can avoid objects with minimum lateral jerk value. Thus, the longitudinal distance is depends on lateral maximum shift length, lateral jerk constraints and current ego speed. Additionally, it has to consider the distance used for the preparation phase.
```

Comment:
```suggestion
If the parameter `detection_area.static` is set to `false`, the module creates a detection area so that the ego can avoid objects with minimum lateral jerk value. Thus, the longitudinal distance depends on maximum lateral shift length, lateral jerk constraints and current ego speed. Additionally, it has to consider the distance used for the preparation phase.
```